src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter.UnitTest/UnitHandlerTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Globalization;
+using NUnit.Framework;
+
+namespace Community.PowerToys.Run.Plugin.UnitConverter.UnitTest
+{
+    [TestFixture]
+    public class UnitHandlerTests {
+        [TestCase(new string[] { "1", "meter", "in", "centimeter" }, UnitsNet.QuantityType.Length, UnitHandler.Abbreviated.Neither)]
+        [TestCase(new string[] { "1", "m", "in", "cm" }, UnitsNet.QuantityType.Length, UnitHandler.Abbreviated.Both)]
+        [TestCase(new string[] { "1", "m", "in", "centimeter" }, UnitsNet.QuantityType.Length, UnitHandler.Abbreviated.First)]
+        [TestCase(new string[] { "1", "meter", "in", "cm" }, UnitsNet.QuantityType.Length, UnitHandler.Abbreviated.Second)]
+        public void ParsesInputForAbbreviations(string[] input, UnitsNet.QuantityType qType, UnitHandler.Abbreviated expectedResult) {
+            (UnitHandler.Abbreviated abbreviated, UnitsNet.QuantityInfo quantityInfo) result = UnitHandler.ParseInputForAbbreviation(input, qType);
```

Comment:
Same q below, is stylecop / fxcop enabled here? this should have been flagged as it is a Java / JS style brace, not c#

Reply:
I missed adding stylecop, but it's been added now